docs/service-mesh/eventing/README.md

```diff
@@ -0,0 +1,68 @@
+## Installing components with Service Mesh
+
+```shell
+make images install-strimzi install-kafka-with-mesh install-serving-with-mesh
+```
+
+## Manually testing basic cases
+
+### Setup resources
+
+```shell
+dir=docs/service-mesh/eventing
+# Create Kafka Broker
+kubectl apply -n serverless-tests -f "${dir}/resources/kafka-broker-example.yaml"
+# Create KafkaNamespaced Broker
+kubectl apply -n serverless-tests -f "${dir}/resources/kafka-broker-namespaced-example.yaml"
+# Create PingSource
+kubectl apply -n serverless-tests -f "${dir}/resources/ping-source.yaml"
+# Create KafkaSource and KafkaSink
+kubectl apply -n kafka -f "${dir}/resources/kafka-source-topic.yaml"
+kubectl apply -n serverless-tests -f "${dir}/resources/kafka-source-example.yaml"
+kubectl apply -n serverless-tests -f "${dir}/resources/kafka-sink-example.yaml"
+kubectl apply -n serverless-tests -f "${dir}/resources/ping-source-kservice.yaml"
+```
+
+Wait for them to become ready by running:
+
+```shell
+kubectl get all -n serverless-tests
+```
+
+### Using a custom source
+
+Run the custom source with Istio's proxy injected:
+
+```shell
+kubectl -n serverless-tests run curl --annotations=sidecar.istio.io/inject=true --image=radial/busyboxplus:curl -i --tty --rm
```

Comment:
just curious (and no need to change, since `curl` is prefectly fine), but would the contain behind `kn event send` be able to do this too? Or should we add support for the istio labeling part?

Reply:
I'm not sure, I've not checked if kn event supports annotating the pod created for the in cluster sender